src/reference/asciidoc/aggregator.adoc

```diff
@@ -89,66 +81,54 @@ public AbstractCorrelatingMessageHandler(MessageGroupProcessor processor, Messag
     ...
 }
 ----
+====
 
-As for actual processing of the message group, the default implementation is the `DefaultAggregatingMessageGroupProcessor`.
-It creates a single Message whose payload is a List of the payloads received for a given group.
-This works well for simple Scatter Gather implementations with either a Splitter, Publish Subscribe Channel, or Recipient List Router upstream.
+As for the actual processing of the message group, the default implementation is the `DefaultAggregatingMessageGroupProcessor`.
+It creates a single `Message` whose payload is a `List` of the payloads received for a given group.
+This works well for simple scatter-gather implementations with a splitter, a publish-subscribe channel, or a recipient list router upstream.
```

Comment:
The convention is to not add emphasis. Capitalization is a form of emphasis. The structure of the sentence should make clear what's important and what's not. So, yes, I de-capitalize everything that isn't a title (and titles get formatted - italics for works and curly quotation marks for parts of works).

We could add a note to the Code Conventions section indicating that we capitalize pattern names. I would still prefer not to, but I could live with that. 

Possibly interesting aside: In the 18th century (and later in other languages, such as German), the convention was to capitalize all nouns. English moved away from that in the early 1800s (coinciding with the Romantic movement). So now the convention is to capitalize only titles (and that only because the titles are themselves capitalized). Times change.